cmd/health.go

```diff
@@ -0,0 +1,94 @@
+package cmd
+
+import (
+	"encoding/json"
+	"github.com/humio/cli/api"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+func newHealthCmd() *cobra.Command {
+	var (
+		textFlag       bool
+		rawFlag       bool
+		versionFlag    bool
+		uptimeFlag     bool
+		failFlag       bool
+		warnAsDownFlag bool
+		selectChecks   []string
+	)
+
+	cmd := &cobra.Command{
+		Use:   "health",
+		Short: "Health",
+		Args:  cobra.ExactArgs(0),
+
+		Run: func(cmd *cobra.Command, args []string) {
+			client := NewApiClient(cmd)
+
+			if textFlag {
+				healthStr, err := client.HealthString()
+
+				exitOnError(cmd, err, "error getting health information")
+
+				cmd.Println(healthStr)
+
+				return
+			}
+
+			health, err := client.Health()
+			exitOnError(cmd, err, "error getting health information")
+
+			if rawFlag {
+				_, _ = cmd.OutOrStdout().Write(health.Json())
+				return
+			}
+
+			if versionFlag {
```

Comment:
Does it make sense to both print raw and the version?

Reply:
You either print the raw json, the version, the uptime or something else.
It might be useful somewhere to have easy access to the version in a script or something, without needing jq or similar.

Reply:
I did not see the return statement in each 👍